tests/unit/llms/providers/test_watsonx.py

```diff
@@ -142,3 +142,38 @@ def test_params_replace_default_values_with_none(provider_config):
     # check default value overrided by None
     assert GenParams.DECODING_METHOD in watsonx.params
     assert watsonx.params[GenParams.DECODING_METHOD] is None
+
+
+@patch("ols.src.llms.providers.watsonx.WatsonxLLM", new=WatsonxLLM())
+def test_generic_parameter_mappings(provider_config):
+    """Test generic parameter mapping to provider parameter list."""
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    # some non default values
+    generic_llm_params = {
+        "min_new_tokens": 100,
+        "max_new_tokens": 200,
+        "top_k": 10,
+        "top_p": 1.5,
+        "temperature": 42.0,
```

Comment:
use constants if you agree to create them

Reply:
updated @bparees